src/components/MyAccount/NewSettings/NewAccountSettingsTab.tsx

```diff
@@ -1,44 +1,95 @@
-import { Flex, Banner } from "@nypl/design-system-react-components"
-import { useContext, useState } from "react"
+import { Flex, Banner, Link, Text } from "@nypl/design-system-react-components"
+import { useContext, useEffect, useRef, useState } from "react"
 import { PatronDataContext } from "../../../context/PatronDataContext"
-import PhoneEmailForm from "./PhoneEmailForm"
+import SettingsInputForm from "./SettingsInputForm"
+import SettingsSelectForm from "./SettingsSelectForm"
+import PasswordForm from "./PasswordForm"
 
 const NewAccountSettingsTab = () => {
   const {
-    updatedAccountData: { patron },
+    updatedAccountData: { patron, pickupLocations },
   } = useContext(PatronDataContext)
-  const [isSuccess, setIsSuccess] = useState(false)
-  const [isFailure, setIsFailure] = useState(false)
+  const [status, setStatus] = useState<[string, string?]>(["none"])
```

Comment:
Alrighty... I did not consider that having the other error message was going to add another dimension to the success/failure state. Now that I see this added dimension, I see there are in fact two states needed to be kept track of (failure/success, and whether the failure is a current password failure). Could you try and rethink another approach to this banner situation? Don't worry about my specific directive before to collapse status into one state, unless it can end up being truly one state.